README.md

```diff
@@ -53,7 +53,7 @@ $ floaty --help
 Grabbing a token for authenticated pooler requests:
 
 ```
-floaty token get --user username --url https://vmpooler.example.net/api/v1
+floaty token get --user username --url https://vmpooler.example.net/api/v1 --api (vmpooler|abs)
```

Comment:
Did you actually add this `--api` flag?
```
➜  vmfloaty git:(add_abs_vm_get) ✗ be bin/floaty token get --user brandon.high --url https://cinext-abs-prod.delivery.puppetlabs.net --api abs

invalid option: --api
```

Reply:
No, that was a previous iteration before I realized there was a "service" feature.